examples/simple-regional-cluster/main.tf

```diff
@@ -5,5 +5,8 @@ module "gke" {
   regional   = true
   region     = var.region
 
+  node_auto_upgrade = false
+  node_auto_repair  = false
```

Comment:
I think we can keep the autor-repair feature on
`If GKE detects that a node requires repair, the node is drained and re-created. GKE waits one hour for the drain to complete. If the drain doesn't complete, the node is shut down and a new node is created.`
It will just recreate the node